src/Client/Request.php

```diff
@@ -43,7 +44,7 @@ public function action(): string
             return Str::of($contentType)->afterLast('action=')->remove('"');
         }
 
-        return $this->request->getHeaderLine('SOAPAction');
+        return Str::of($this->request->getHeaderLine('SOAPAction'))->remove('"');
```

Comment:
You could use the `\Soap\Psr18Transport\HttpBinding\SoapActionDetector` for this ;)